src/style.css

```diff
@@ -2,0 +1,67 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.search-bar {
+  font-family: Avenir, sans-serif;
+  display: block;
+  width: 100%;
+  margin-top: 20px;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  outline: none;
+}
+
+.search-bar::placeholder {
+  color: #3d4e61;
+}
+
+.search-bar__big {
+  height: 70px;
+  padding-left: 62px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px;
+  background-position: left 26px center;
+  background-color: white;
+
+  font-size: 16px;
+  font-weight: 300;
+}
+
+.search-bar__small {
+  height: 42px;
+  padding-left: 33px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: left 13px center;
+  background-size: 11px 11px;
+  background-color: white;
+
+  font-size: 14px;
+  font-weight: 300;
```

Comment:
Similar to the '.search-bar__big', the font-weight '300' is used for '.search-bar__small'. Ensure that this weight is supported by the font file.